packages/caravan-health/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@caravan/health",
+  "version": "1.0.0",
```

Comment:
let's mark this as `1.0.0-beta` to make it clear that it might still undergo changes.